libcxx/docs/Status/Cxx23Issues.csv

```diff
@@ -241,9 +241,9 @@
 "`3817 <https://wg21.link/LWG3817>`__","Missing preconditions on ``forward_list`` modifiers", "November 2022","","",""
 "`3818 <https://wg21.link/LWG3818>`__","Exposition-only concepts are not described in library intro", "November 2022","|Nothing to do|","",""
 "`3822 <https://wg21.link/LWG3822>`__","Avoiding normalization in ``filesystem::weakly_canonical``", "November 2022","","",""
-"`3823 <https://wg21.link/LWG3823>`__","Unnecessary precondition for ``is_aggregate``", "November 2022","","",""
+"`3823 <https://wg21.link/LWG3823>`__","Unnecessary precondition for ``is_aggregate``", "November 2022","|Nothing To Do|","",""
```

Comment:
This is handled by the Clang frontend (13c32288354b6175527f4fdece623157d3637da3). Should we add a note as one may be confused that while this entry says "Nothing To Do", `std::is_aggregate_v<Incomplete[2]>` only starts to compile since Clang 16 ([Godbolt link](https://godbolt.org/z/deb3qo3Kz))?

Reply:
Since we only support Clang 17 and newer I don't feel we need to add a note.